### PR TITLE
[Durable Objects] Docs enhancement part 1

### PR DESCRIPTION
**Pull request by Oxyjun**
### Summary

~~1. Improved the **Overview page**.
    - Added "What is a Durable Object?". Lifted paragraphs from [this blog post](https://blog.cloudflare.com/sqlite-in-durable-objects/#refresher-what-are-durable-objects).
    - Added "Utilize Durable Objects". Lifted one paragraph from the blog post.~~
    - Added "Get started" button.
2. Added **glossary tooltips** across the docs.
3. **Moved Changelog** out of "Platform", and into its own item.
4. **Moved up REST API** link to sit under "Runtime API".
5. **Moved Glossary** into "Reference".

The full restructure can be tackled in a separate PR to even out the review burden.

### Documentation checklist

- [ ] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.
- [ ] If a larger change - such as adding a new page- an issue has been opened in relation to any incorrect or out of date information that this PR fixes.
- [ ] Files which have changed name or location have been allocated [redirects](https://developers.cloudflare.com/pages/configuration/redirects/#per-file).

### CODE REVIEW

File: src/content/docs/durable-objects/index.mdx [hunk 1]
```diff
@@ -11,22 +11,30 @@ head:
...
-<Plan type="paid" />
+## Utilize Durable Objects
 
 Durable Objects provide a powerful compute API for coordinating multiple clients or users. Each Durable Object has private, transactional and strongly consistent storage attached.
 
+They are especially good at real-time collaboration, but are not limited to this use case. They are general-purpose servers that can implement any logic you desire to serve requests. More generally, Durable Objects are a basic building block for distributed systems.
+
 Use Durable Objects to build collaborative editing tools, interactive chat, multiplayer games and applications that need coordination among multiple clients, without requiring you to build serialization and coordination primitives on your own.
 
+<LinkButton href="/durable-objects/get-started/walkthrough/">Get started</LinkButton>
```
Comment (vy-ton) [inline src/content/docs/durable-objects/index.mdx:26]: Too much text before user arrives at `Get started` button

File: src/content/docs/durable-objects/index.mdx [hunk 2]
```diff
@@ -11,22 +11,30 @@ head:
...
+## What are Durable Objects?
 
+Durable Objects are a part of the Cloudflare Workers serverless platform. A Durable Object is essentially a small server that can be addressed by a unique name and can keep state both in-memory and on-disk. Workers running anywhere on Cloudflare's network can send messages to a Durable Object by its name, and all messages addressed to the same name — from anywhere in the world — will find their way to the same Durable Object instance.
 
-</Description>
+Durable Objects are intended to be small and numerous. A single application can create billions of Durable Objects distributed across our global network. Cloudflare automatically decides [where a Durable Object should live](/durable-objects/reference/data-location/) based on where it is accessed, automatically starts it up as needed when requests arrive, and shuts it down when idle. A Durable Object has in-memory state while running and can also optionally store long-lived durable state. Since there is exactly one Durable Object for each name, a Durable Object can be used to coordinate between operations on the same logical object.
```
Comment (vy-ton) [inline src/content/docs/durable-objects/index.mdx]: This is too much detail for the `Overview` page. Should have a 1-2 sentence description. 

Can we remove Overview copy text from this PR? Let's start a draft in a google doc so we land the messaging

**Review by vy-ton — CHANGES_REQUESTED** [review]

File: src/content/docs/durable-objects/index.mdx (hunk 2 above)
Comment (Oxyjun) [reply src/content/docs/durable-objects/index.mdx]: Understood! Reverted the wording (but have left the "Get started" button, I think that's a nice addition).

**Review by vy-ton — APPROVED** [review]

File: src/content/glossary/durable-objects.yaml [hunk 3]
```diff
@@ -11,7 +11,7 @@ entries:
 
 - term: "Durable Object"
   general_definition: |-
-    An individual instance of a Durable Object class. A Durable Object is globally unique (referenced by ID), provides a global point of coordination for all methods/requests sent to it, and has private, persistent storage that is not shared with other Durable Objects within a namespace.
+    A Durable Object is an individual instance of a Durable Object class. A Durable Object is globally unique (referenced by ID), provides a global point of coordination for all methods/requests sent to it, and has private, persistent storage that is not shared with other Durable Objects within a namespace.
```
Comment (Maddy-Cloudflare) [inline src/content/glossary/durable-objects.yaml:14]: There is an opportunity here to link this page: https://developers.cloudflare.com/durable-objects/api/id/

**Review by Maddy-Cloudflare — APPROVED** [review]

File: src/content/glossary/durable-objects.yaml (hunk 3 above)
Comment (Oxyjun) [reply src/content/glossary/durable-objects.yaml:14]: Ah thanks for that suggestion! In this particular instance I think I'd prefer to leave it as a short definition without linking out, but I'll keep an eye out to see if there are other sections where I might be able to cross-reference :D

**Pull request merged**